public/js/index.js

```diff
@@ -437,7 +437,17 @@ Index.checkVersion = function () {
     const githubAPI = "https://api.github.com/repos/difegue/lanraragi/releases/latest";
 
     fetch(githubAPI)
-        .then((response) => response.json())
+        .then((response) => {
+            if (response.ok) {
+                return response.json();
+            }
+            if (response.status === 403) {
+                console.error("API rate limit exceeded: ", response);
```

Comment:
Probably worth mentioning Github in those messages too for endusers.

Reply:
And since I merged JS translations I guess you get to make those localizable now!